queries.yaml

```diff
@@ -985,6 +985,42 @@ post_process:
       properties:
         - walking_network
         - walking_shield_text
+  # drop RCN network & shield text below zoom 13.
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 13
+      properties:
+        - bicycle_network
```

Comment:
Reviewing https://github.com/tangrams/walkabout-style/blob/gh-pages/layers/bike-interlay.yaml I see that the network IS referenced like `bicycle_network: [icn,ncn,rcn]` at lower zooms, but `rcn` is ignored at super low zooms like `filter: { bicycle_network: rcn, $zoom: [8,9,10] }`:

https://github.com/tangrams/walkabout-style/blob/gh-pages/layers/bike-interlay.yaml#L1788-L1797

So it would be:

- okay to drop `bicycle_network` from 0 to 11 (or is that 10, I get confused)
- okay to drop `bicycle_shield_text` from 0 to 12 (one more zoom than `bicycle_network` is dropped)
- see other comment about always dropping the `all*` variants until the max zoom) even for `rcn`